internal/test/issues/issue-1039/issue_test.go

```diff
@@ -0,0 +1,282 @@
+package issue1039
+
+import (
+	_ "embed"
+	"encoding/json"
+	issue1039nullable "github.com/deepmap/oapi-codegen/v2/internal/test/issues/issue-1039/nullable-disabled"
+	"github.com/deepmap/oapi-codegen/v2/pkg/codegen"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+//go:embed spec.yaml
+var spec []byte
+
+func ToStringPointer(str string) *string {
+	return &str
+}
+
+func TestNullableDisabled(t *testing.T) {
```

Comment:
I'd also recommend moving this into the package with the default behaviour, or making it clearer what this is - it was a little confusing reading this one first, then seeing the new behaviour - maybe we should keep this next to the existing behaviour's generated code?